cmd/flags.go

```diff
@@ -206,3 +227,20 @@ func EnvOrDefault(key string, defaultVal string) string {
 	}
 	return defaultVal
 }
+
+func updatePath(pathToUpdate *string, defaultPath, platformAPI, layersDir string) {
+	if *pathToUpdate != defaultPath {
+		return
+	}
+	fileName := filepath.Base(defaultPath)
+	if isPlatformAPILessThan05(platformAPI) || layersDir == "" {
+		// layersDir is unset when this call comes from the rebaser - will be fixed as part of https://github.com/buildpacks/spec/issues/156
+		*pathToUpdate = filepath.Join(".", fileName)
+	} else {
+		*pathToUpdate = filepath.Join(layersDir, fileName)
+	}
+}
+
+func isPlatformAPILessThan05(platformAPI string) bool {
+	return api.MustParse(platformAPI).Compare(api.MustParse("0.5")) < 0
+}
```

Comment:
Given that `api.MustParse(platformAPI).Compare(api.MustParse("0.5")) < 0` is already a one-liner I don't see the value of this function unless we rename it to something like `shouldUseWorkingDirRelativeDefaults`.

Reply:
You're right. When I created this function, I called it multiple times around the code but after some refactoring I did (before making the PR), it's being called only once.
I'll delete it.
Thanks!